<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>librariandeletebook</class>
 <widget class="QDialog" name="librariandeletebook">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>874</width>
    <height>425</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>删除书籍</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(158, 216, 255);</string>
  </property>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>200</y>
     <width>241</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;请输入删除书籍id：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>360</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>90</y>
     <width>311</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 15pt &quot;等线&quot;;</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;请输入删除书籍id：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>190</y>
     <width>791</width>
     <height>161</height>
    </rect>
   </property>
   <row>
    <property name="text">
     <string/>
    </property>
   </row>
   <column>
    <property name="text">
     <string>书籍id</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>书籍名称</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>书籍作者</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>借阅状态</string>
    </property>
   </column>
  </widget>
  <widget class="QLineEdit" name="lineEdit_4">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>140</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>140</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;等线&quot;;</string>
   </property>
   <property name="text">
    <string>查询</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>0</y>
     <width>181</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(48, 60, 175);
font: 87 9pt &quot;Arial&quot;;
font: 14pt &quot;Arial&quot;;
font: 10pt &quot;华文新魏&quot;;
font: 11pt &quot;Arial&quot;;
font: 11pt &quot;华文新魏&quot;;</string>
   </property>
   <property name="text">
    <string>删除书籍</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
